### PR TITLE
Migrated test_client_side_quorum_with_fixed_validate_max_bricks

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py)

Updates: #292

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
```sh
Results for afr
Test nature : disruptive
TestName : test_client_side_quorum_with_fixed_validate_max_bricks
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:41       |      NA     |
|   dist-rep  |     PASS    |        00:00:54       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 01 minutes 53 seconds
```

File: tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        * Above step should fail
+        """
+        # set cluster.quorum-type to fixed
+        options = {"cluster.quorum-type": "fixed"}
+        redant.set_volume_options(self.vol_name, options,
+                                  self.server_list[0])
+        # get the subvolumes
+        subvols_list = redant.get_subvols(self.vol_name, self.server_list[0])
+        redant.logger.info(f"Number of subvolumes in volume {self.vol_name}"
+                           f" is {len(subvols_list)}")
```
Comment (nik-redhat) [inline tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py:46]: nit not needed

File: tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+        # get the number of bricks in replica set
+        num_bricks_in_subvol = len(subvols_list[0])
+        redant.logger.info("Number of bricks in each replica "
+                           f"set: {num_bricks_in_subvol}")
```
Comment (nik-redhat) [inline tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py:51]: Not needed

File: tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        # set cluster.quorum-count to higher value than the number
+        # of bricks in replica set
+        start_range = num_bricks_in_subvol + 1
+        end_range = num_bricks_in_subvol + 30
+        for i in range(start_range, end_range):
+            options = {"cluster.quorum-count": f"{i}"}
+            ret = redant.set_volume_options(self.vol_name, options,
+                                            self.server_list[0],
+                                            excep=False)
+            if ret['msg']['opRet'] != '-1':
```
Comment (nik-redhat) [inline tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py:62]: A safer check would be `== '0'`.

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py:46]: Ack

File: tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py (hunk 2 above)
Comment (aujjwal-redhat) [reply tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py:51]: Ack

File: tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py (hunk 3 above)
Comment (aujjwal-redhat) [reply tests/functional/afr/test_client_side_quorum_with_fixed_validate_max_bricks.py:62]: Ack

**Comment by aujjwal-redhat** [comment]
```sh
Test nature : disruptive
TestName : test_client_side_quorum_with_fixed_validate_max_bricks
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:43       |      NA     |
|   dist-rep  |     PASS    |        00:00:50       |      NA     |
+-------------+-------------+-----------------------+-------------+
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**